src/styles/markdown.scss

```diff
@@ -99,13 +99,13 @@
     }
 
     &.tip {
-      background-color: #DCF2FD;
-      color: #618ca0;
+      background-color: #eaf8ff;
```

Comment:
Can you post a screen shot with this changes? For reviewers is hard sometimes to infer the impact of the change and checking out and building the branch is a little time consuming, thanks :)

Reply:
Here are screenshots with my changes. The changes are very subtle though :)
![localhost_3000_ 1](https://user-images.githubusercontent.com/10567153/41506714-b4309e7c-722b-11e8-88f4-5770906b87dc.png)
![localhost_3000_concepts_](https://user-images.githubusercontent.com/10567153/41506715-b6f2b53c-722b-11e8-86e4-38f9af49f99c.png)